packages/components/src/components/field/field.tsx

```diff
@@ -0,0 +1,21 @@
+import classNames from 'classnames';
+import React, { ReactNode } from 'react';
+
+type TFieldProps = {
+    className?: string;
+    message: string | ReactNode | boolean;
```

Comment:
@iman-fs Seems like the `message` is always `string` 🤔 Please check this one too 🙏🏻

Reply:
as I checked we have ReactNode too! but boolean doesn't make sense!

Reply:
ReactNode type covers all of the following types and there's no need to repeat its covered types over here.

(string | number | boolean | React.ReactElement<any, string | React.JSXElementConstructor<any>> | React.ReactFragment | React.ReactPortal | null | undefined)

Reply:
@hirad-rewok That would be too loose in my opinion, We should be more strict on types to have a safer codebase.

Reply:
I checked it too. string is enough for `message`. thanks guys